ci/docker/base.ml.Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM $DOCKER_IMAGE_BASE_TEST
 
 COPY . .
 
+RUN apt-get install -y -qq maven openjdk-8-jre openjdk-8-jdk
```

Comment:
need this to build java

Reply:
how much size will this add? I recall that the size of maven and jdk is non-trivial actually. should we consider split java required stuff into another base? (we can reuse this build file)

Reply:
I can make it so that we install this only for the data.mongodb build. There is only one data test requires java to be installed.